docs/vNext/tutorials_fabric_existing.md

```diff
@@ -0,0 +1,724 @@
+            },
+            "grpcOptions": {
+                "ssl-target-name-override": "peer0.org1.example.com",
+                "hostnameOverride": "peer0.org1.example.com"
+            }
+        }
+    },
+    "certificateAuthorities": {
+        "ca.org1.example.com": {
+            "url": "https://localhost:7054",
+            "caName": "ca-org1",
+            "tlsCACerts": {
+                "pem": "-----BEGIN CERTIFICATE-----<UNIQUE CONTENT>-----END CERTIFICATE-----\n"
+            },
+            "httpOptions": {
+                "verify": false
+            }
+        }
+    },
+}
+```
+
+## Step 4 - Test Workload Module
+
+The workload module interacts with the deployed smart contract during the benchmark round. The workload module extends the Caliper class `WorkloadModuleBase` from `caliper-core`. The workload module provides three overrides: 
+- `initializeWorkloadModule` - used to initialise any required items for the benchmark
+- `submitTransaction` - used to interact with the smart contract method during the monitored phase of the benchmark
+- `cleanupWorkloadModule` - used to clean up after the completion of the benchmark
+
+For more information, please see the specific documentation on `Workload Configuration` accessible on the left hand menu.
+
+The workload we will be driving aims to benchmark the querying of existing cars within the world state database. Consequently we will use all three phases available in the workload module:
+- `initializeWorkloadModule` - to create cars that may be queried in the `submitTransaction` phase
+- `submitTransaction` - to query cars created in the `initializeWorkloadModule` phase
+- `cleanupWorkloadModule` - used to remove cars created in the `initializeWorkloadModule` phase so that the benchmark may be repeated
+
+### Create The Workload Module
+
+Under the **benchmarks** folder create a subfolder called **workload**, and navigate to this folder.
+1. Run `npm init` to create a `package.json` file. Open the file and add `"@hyperledger/caliper-core": "0.4.0"` as a dependency.
+2. Run `npm install` to obtain the local package requirement.
+3. Create a file called **queryAssetBenchmark.js**. This is the file we will be populating. When we populate this file we make reference to the deployed smart contract **fabcar.js** file that can be found in: **fabric-samples** -> **chaincode** -> **fabcar** -> **javascript** -> **lib** -> **fabcar.js**
+
+### Setting up File
+
+Create the workload file template: 
+
+``` bash
+'use strict';
+
+const { WorkloadModuleInterface } = require('@hyperledger/caliper-core');
+
+class MyWorkload extends WorkloadModuleInterface {
+    constructor() {
+        super();
+    }
+    
+    async initializeWorkloadModule(workerIndex, totalWorkers, roundIndex, roundArguments, sutAdapter, sutContext) {
+        await super.initializeWorkloadModule(workerIndex, totalWorkers, roundIndex, roundArguments, sutAdapter, sutContext);
```

Comment:
need to remove this line otherwise you get an error saying you need to implement `initializeWorkloadModule`